index.html

```diff
@@ -4,12 +4,580 @@
+            <h2 class="cta-heading">Join Our Digital
+                Creative Revolution</h2>
+            <p class="cta-sub">Share your talent with the world and become a part of our ever-growing community of
+                creators.</p>
+            <button type="button" class="discord-button">Become Acreator</button>
+        </section>
+        <section class="blog">
+            <h2 class="blog-heading">Latest Expert Design Insights</h2>
+            <p class="blog-label">BLOG</p>
+            <div class="next-previous-buttons">
+                <a href="#" class="previous round">&#8249;</a>
+                <a href="#" class="next round">&#8250;</a>
+            </div>
+            <ul class="blog-list">
+                <li class="blog-list-item">
+                    <article class="blog-article">
+                        <figure class="article-figure">
+                            <img src="#" alt="photo">
+                            <p class="figure-tag">TYPOGRAPHY</p>
+                            <figcaption class="article-caption">
+                                <h3 class="article-title">Mastering the Art of Typography: Tips for Eye-Catching Designs
+                                </h3>
+                                <p class="article-date">5 APRIL 2024</p>
+                            </figcaption>
+                        </figure>
+                    </article>
+                </li>
+                <li class="blog-list-item">
+                    <article class="blog-article">
+                        <figure class="article-figure">
+                            <img src="#" alt="photo">
+                            <p class="figure-tag">UI/UX DESIGN</p>
+                            <figcaption class="article-caption">
+                                <h3 class="article-title">The Ultimate Guide to UI/UX: Enhancing App User Experience
+                                    with Design</h3>
+                                <p class="article-date">16 MARCH 2024</p>
+                            </figcaption>
+                        </figure>
+                    </article>
+                </li>
+                <li class="blog-list-item">
+                    <article class="blog-article">
+                        <figure class="article-figure">
+                            <img src="#" alt="photo">
+                            <p class="figure-tag">DESIGN TRENDS</p>
+                            <figcaption class="article-caption">
+                                <h3 class="article-title">2024’s Top Graphic Design Trends: What Would Be Shaping the
+                                    Future of Design?</h3>
+                                <p class="article-date">27 FEBRUARY 2024</p>
+                            </figcaption>
+                        </figure>
+                    </article>
+                </li>
+            </ul>
+        </section>
+    </main>
+    <footer class="page-footer">
 
-    
+    </footer>

```

Comment:
The footer tag is closed too early. Ensure all elements are properly nested and closed. (: